src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -30,6 +30,8 @@
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --enable-private-link true --private-link-scope-resource-id pls/resource/arm/id
     - name: Onboard a connected kubernetes cluster with custom onboarding timeout.
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --onboarding-timeout 600
+    - name: Onboard a connected kubernetes cluster with least privileges
+      text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --config-settings "{\\"service-account-name\\":\\"azure-arc-operator-sa\\"}" --least-privilege true
```

Comment:
would be good to add this help example in upgrade & delete

Reply:
Discussed offline, upgrade and delete commands usage will be same as regular mode and hence not adding examples for that